samples/step-by-step/Core_5/Client/Program.cs

```diff
@@ -3,24 +3,39 @@
 
 class Program
 {
-
+    #region StepByStep-ClientInit
     static void Main()
     {
+        // This makes it easier to tell console windows apart
```

Comment:
I think it will keep the code snippet much cleaner, if the comments are part of the documentation that describes the Client and how to setup the client than individual line comments in the code. @Particular/docs-maintainers - thoughts?